utils/cloudfront.go

```diff
@@ -186,13 +179,29 @@ func (d *Distribution) setDistributionConfigDefaults(config *cloudfront.Distribu
 		aws.String("PATCH"),
 		aws.String("DELETE"),
 	}
-
-	config.PriceClass = aws.String("PriceClass_100")
 }
 
 func (d *Distribution) Create(callerReference string, domains []string, origin, path string, defaultTTL int64, insecureOrigin bool, forwardedHeaders Headers, forwardCookies bool, tags map[string]string) (*cloudfront.Distribution, error) {
 	distConfig := new(cloudfront.DistributionConfig)
+
+	distConfig.DefaultCacheBehavior = &cloudfront.DefaultCacheBehavior{
+		ForwardedValues: &cloudfront.ForwardedValues{
+			Cookies:              &cloudfront.CookiePreference{},
+			QueryStringCacheKeys: &cloudfront.QueryStringCacheKeys{},
+		},
+		LambdaFunctionAssociations: &cloudfront.LambdaFunctionAssociations{
+			Quantity: aws.Int64(0),
+		},
+		TrustedSigners: &cloudfront.TrustedSigners{},
+		AllowedMethods: &cloudfront.AllowedMethods{
+			CachedMethods: &cloudfront.CachedMethods{},
+		},
+	}
+
 	d.setDistributionConfigDefaults(distConfig)
+	d.setDistributionConfigCacheBehaviours(distConfig)
+	d.setDistributionConfigDefaultCacheBehaviour(distConfig)
```

Comment:
Could this function include the job being done on line 187?

Reply:
`CacheBehaviors` and `DefaultCacheBehavior` are different parts of the `DistributionConfig` struct which is why I kept them separate

Happy to merge them though